Userland/Libraries/LibWeb/HTML/CanvasRenderingContext2D.cpp

```diff
@@ -376,7 +376,7 @@ void CanvasRenderingContext2D::fill(Path2D& path, StringView fill_rule)
 
 JS::GCPtr<ImageData> CanvasRenderingContext2D::create_image_data(int width, int height) const
 {
-    return ImageData::create_with_size(realm(), width, height);
+    return MUST(ImageData::create(realm(), width, height));
```

Comment:
This feels like one of the few APIs where we *shouldn't* blindly `MUST` our way through.

If someone requests a monstrously huge `ImageData`, should we not throw instead of crashing? What do other browsers do?

Reply:
Absolutely, other browsers propagate the exception, done the same for us.